modules/ROOT/pages/_partials/eu-cloud-access.adoc

```diff
@@ -0,0 +1,17 @@
+= To Access the EU Control Plane (Anypoint Studio)
+
+You can configure your Anypoint Studio 6.4.2 and later to point to the EU control plane.
+
+This process enables you to perform tasks such as deploying to CloudHub, importing an asset from Exchange, or importing an API definition from Design Center, in the EU control plane.
+
+. Open the preferences view. +
+* In MacOS navigate to *Anypoint Studio* > *Preferences..* or hit `cmd + ,`.
```

Comment:
```suggestion
* In MacOS, either navigate to *Anypoint Studio* > *Preferences...* or press `cmd + ,`.
```

"Preferences has just two dots .. or three ...?

GLOBALLY replace "hit" with "press", and clarify exactly which keys the user is pressing. These are currently formatted as syntax parameters, but they should be represented as UI controls/keycaps: "or press the *control* key simultaneously with the *,* key"?